wiki/Placeholders.md

```diff
@@ -1460,7 +1460,7 @@ Available gamemodes:
 ```
 ----
 
-- ### **[Clans](https://www.spigotmc.org/resources/78415/)**
+- ### **[ClansFree](https://www.spigotmc.org/resources/78415/)**
```

Comment:
The list at the top (https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#c) also needs to be updates, since you changed the Header here.

Just change `- [Clans](#clans-1)` to `[Clans](#clansfree)`

Reply:
I think i did it!